### PR TITLE
NOISSUE: Update Docs

### PR DESCRIPTION
**Pull request by WashingtonKK**
This updates cocos docs to factor in some of the recent changes.

### CODE REVIEW

File: docs/cli.md [hunk 1]
```diff
@@ -12,33 +12,112 @@ To build the CLI, follow these steps:
...
 
+Currently, support is provided for two types of algorithms: executable binaries and python files. The above command expects an algorithm in the binary format which will be executed inside agent. For python files, the algo file is required, along with the requirements file and the python runtime. To run a python file, use the following command:
+
+```bash
+export AGENT_GRPC_URL=<agent_host:agent_host>
+
+./build/cocos-cli algo /path/to/algorithm /path/to/private/key --algorithm python --requirements /path/to/requirements.txt --python-runtime python
+```
+
+The agent grpc url is required for this operation, this will be available once the TEE has been provisioned and agent is running.
```
Comment (SammyOina) [inline docs/cli.md:47]: move this and export to the beginning so you don't have to repeat

File: docs/cli.md [hunk 2]
```diff
@@ -12,33 +12,112 @@ To build the CLI, follow these steps:
...
+
+### Fetch and Validate Attestation Report
+
+To fetch or validate the attestation report, use the following commands.
+
+To fetch attestation report:
+```bash
+./build/cocos-cli attestation get report <report_data>
```
Comment (SammyOina) [inline docs/cli.md:81]: this is not correct

File: docs/cli.md [hunk 3]
```diff
@@ -12,33 +12,112 @@ To build the CLI, follow these steps:
...
+```
+
+To validate attestation report
+```bash
+./build/cocos-cli attestation validate <attestation_report_file_path>
```
Comment (SammyOina) [inline docs/cli.md:86]: report data flag is compulsory, also document optional flags here

File: docs/developer-guide.md [hunk 4]
```diff
@@ -95,21 +97,31 @@ Copy `rootfs.cpio.gz` and `bzImage` from the buildroot output directory files to
...
 cd cmd/manager
-MANAGER_GRPC_URL=localhost:7001 MANAGER_LOG_LEVEL=debug MANAGER_QEMU_USE_SUDO=false  MANAGER_QEMU_ENABLE_SEV=false MANAGER_QEMU_OVMF_CODE_FILE=/usr/share/edk2/x64/OVMF_CODE.fd MANAGER_QEMU_OVMF_VARS_FILE=/usr/share/edk2/x64/OVMF_VARS.fd go run main.go
+MANAGER_GRPC_URL=localhost:7001 \
+    MANAGER_LOG_LEVEL=debug \
+    MANAGER_QEMU_USE_SUDO=false \
+    MANAGER_QEMU_ENABLE_SEV=false \
+    MANAGER_QEMU_SEV_CBITPOS=51 \
+    MANAGER_QEMU_OVMF_CODE_FILE=/usr/share/OVMF/OVMF_CODE.fd \
+    MANAGER_QEMU_OVMF_VARS_FILE=/usr/share/edk2/ovmf/OVMF_VARS.fd \
+    MANAGER_QEMU_ENABLE_SEV_SNP=false MANAGER_QEMU_MEMORY_SIZE=16G \
```
Comment (SammyOina) [inline docs/developer-guide.md:107]: use only minimum env required to launch manager, for example remove memory size

File: docs/developer-guide.md [hunk 5]
```diff
@@ -95,21 +97,31 @@ Copy `rootfs.cpio.gz` and `bzImage` from the buildroot output directory files to
...
+    go run main.go
```
Comment (SammyOina) [inline docs/developer-guide.md:108]: use ./build path rather than go run

File: docs/getting-started.md [hunk 6]
```diff
@@ -62,7 +68,15 @@ When manager connects to the computations server, the server then sends a comput
...
 cd cmd/manager
-MANAGER_GRPC_URL=localhost:7001 MANAGER_LOG_LEVEL=debug MANAGER_QEMU_USE_SUDO=false  MANAGER_QEMU_ENABLE_SEV=false MANAGER_QEMU_OVMF_CODE_FILE=/usr/share/edk2/x64/OVMF_CODE.fd MANAGER_QEMU_OVMF_VARS_FILE=/usr/share/edk2/x64/OVMF_VARS.fd go run main.go
+MANAGER_GRPC_URL=localhost:7001 \
+    MANAGER_LOG_LEVEL=debug \
+    MANAGER_QEMU_USE_SUDO=false \
+    MANAGER_QEMU_ENABLE_SEV=false \
+    MANAGER_QEMU_SEV_CBITPOS=51 \
+    MANAGER_QEMU_OVMF_CODE_FILE=/usr/share/OVMF/OVMF_CODE.fd \
+    MANAGER_QEMU_OVMF_VARS_FILE=/usr/share/edk2/ovmf/OVMF_VARS.fd \
+    MANAGER_QEMU_ENABLE_SEV_SNP=false MANAGER_QEMU_MEMORY_SIZE=16G \
```
Comment (SammyOina) [inline docs/getting-started.md:78]: same here

File: docs/getting-started.md [hunk 7]
```diff
@@ -62,7 +68,15 @@ When manager connects to the computations server, the server then sends a comput
...
+    go run main.go
```
Comment (SammyOina) [inline docs/getting-started.md:79]: and here

**Review by SammyOina — CHANGES_REQUESTED** [review]

File: docs/cli.md (hunk 1 above)
Comment (WashingtonKK) [reply docs/cli.md:47]: I added it there since it is not required for all cli functions.

Comment (SammyOina) [reply docs/cli.md:47]: just have at the beginning of the cli examples and write that note that it is only required for certain functions. or rather add a section for env variables used by cli instead

File: docs/cli.md [hunk 8]
```diff
@@ -12,33 +12,147 @@ To build the CLI, follow these steps:
...
+
+To validate attestation report
+```bash
+./build/cocos-cli attestation validate <attestation_report_file_path> --report_data <report_data>
+```
+To validate the report data, the report data flag is compulsory.
+
+Optional Flags:
+```shell
+      --CA_bundles stringArray                  PEM format CA bundles for the AMD product. Combined with contents of cabundle_paths.
```
Comment (SammyOina) [inline docs/cli.md:92]: smillarly add it for other cli commands with flags for example algo

**Review by SammyOina — CHANGES_REQUESTED** [review]

**Review by SammyOina — APPROVED** [review]

**Pull request merged**